### ISSUE TITLE
RakLib slot change

### ISSUE THREAD

**Issue by thebigsmileXD** [ImagicalCorp/ImagicalMine#307]
Latest build:
[19:41:44] [DEBUG]: [RakLib Thread #4048] An E_WARNING error happened: "socket_recvfrom(): unable to recvfrom [10054]: Eine vorhandene Verbindung wurde vom Remotehost g" in "/src/raklib/server/UDPServerSocket" at line 59
[19:41:44] [DEBUG]: #0 /src/raklib/server/SessionManager(158): raklib\server\UDPServerSocket->readPacket()
[19:41:44] [DEBUG]: #1 /src/raklib/server/SessionManager(105): raklib\server\SessionManager->receivePacket()
[19:41:44] [DEBUG]: #2 /src/raklib/server/SessionManager(96): raklib\server\SessionManager->tickProcessor()
[19:41:44] [DEBUG]: #3 /src/raklib/server/SessionManager(84): raklib\server\SessionManager->run()
[19:41:44] [DEBUG]: #4 /src/raklib/server/RakLibServer(235): raklib\server\SessionManager->__construct()
[19:41:44] [DEBUG]: #5 /src/raklib/server/RakLibServer(0): {main}()

I am on win 10 0.13.0, maybe there has anything been changed. But sometimes it works

**Comment by matcracker** [comment]
I can confirm it in Android, I haven't idea of this issue

**Comment by keithkfng** [comment]
0.13.0? Seriously? The update has already come. LOL

**Comment by thebigsmileXD** [comment]
My Store on win 10 crashes and my web is too slow on my iPhone.

**Comment by thebigsmileXD** [comment]
Okay, it works now, but the slots still get selected randomly when scrolling through the hotbar

**Comment by matcracker** [comment]
Yes, slots bar are crazy D:

**Comment by keithkfng** [comment]
Still an issue?

**Comment by robske110** [comment]
Yes, espacially on Windows10 when you scroll your bar will just turn into an disco of random blocks

**Comment by thebigsmileXD** [comment]
Was usefull when i made a rainbow block thingy for a christmas sledge that looses gifts xD
